views/profile/index.ejs

```diff
@@ -1,13 +1,13 @@
-<!DOCTYPE html>
+<!-- <!DOCTYPE html>
```

Comment:
Shouldn't this be deleted, if it is commented anyway?

Reply:
It is I must of missed it sorry!